### PR TITLE
Add AIS try out page

### PR DESCRIPTION
**Pull request by isharailanga**
This PR adds content to the Accounts try out flow according to https://docs.google.com/document/d/1v0ktcwPYfHzjeNlHKLFM3h9btvjJyX1GM4ARvkHBY6c/edit?hl=en&forcehl=1#.

### CODE REVIEW

File: en/docs/try-out/account-information-service-flow.md
Comment (Akila94) [inline en/docs/try-out/account-information-service-flow.md:167]: The toolkit can also support the PAN as an account identifier. But this needs to be handled using a customization when integrating with the banking backend. This information will be included in the "Core Banking Integration" documentation (currently WIP). Hope we can update this doc after the core banking integration doc is published. 

No need to change anything from this doc until core banking integration is published. Just added the comment FYI.

Comment (isharailanga) [reply en/docs/try-out/account-information-service-flow.md:167]: hi @Akila94, noted with thanks. I created this issue to track this doc requirement -https://github.com/wso2/docs-open-banking/issues/567.

**Review by Akila94 — APPROVED** [review]

**Comment by Akila94** [comment]
Please merge this after Akram's approval.

File: en/docs/try-out/account-information-service-flow.md
Comment (aka4rKO) [inline en/docs/try-out/account-information-service-flow.md]: Is it OK to call this the PSD2 API here since then it would be the same for payments flow as well?

File: en/docs/try-out/account-information-service-flow.md
Comment (aka4rKO) [inline en/docs/try-out/account-information-service-flow.md]: Im assuming these are the certs used for UK. But these needs to be changed. Since we have only self-signed certs for Berlin we need to see which certs can be added here.

File: en/docs/try-out/account-information-service-flow.md
Comment (aka4rKO) [inline en/docs/try-out/account-information-service-flow.md]: Use the sample provided in the google doc here because there is no DCR for Berlin

File: en/docs/try-out/account-information-service-flow.md
Comment (aka4rKO) [inline en/docs/try-out/account-information-service-flow.md]: add space on both sides \:

File: en/docs/try-out/account-information-service-flow.md
Comment (aka4rKO) [inline en/docs/try-out/account-information-service-flow.md]: Can remove this "For example, account references"

File: en/docs/try-out/account-information-service-flow.md
Comment (aka4rKO) [inline en/docs/try-out/account-information-service-flow.md]: "The attribute values can be" replace with "The attribute values must be"

File: en/docs/try-out/account-information-service-flow.md
Comment (aka4rKO) [inline en/docs/try-out/account-information-service-flow.md]: Use the sample provided in the google doc here because there is no DCR for Berlin

File: en/docs/try-out/account-information-service-flow.md
Comment (isharailanga) [reply en/docs/try-out/account-information-service-flow.md]: @Akila94

File: en/docs/try-out/account-information-service-flow.md
Comment (isharailanga) [reply en/docs/try-out/account-information-service-flow.md]: will copy the steps to deploy to a common place

File: en/docs/try-out/account-information-service-flow.md
Comment (isharailanga) [reply en/docs/try-out/account-information-service-flow.md]: Fixed in 492e0e859d9d73f0ff9ad20698d7e1ca2e26bf9a & 69359a31b168e4c48b49c82cf618431b74398042

File: en/docs/try-out/account-information-service-flow.md
Comment (isharailanga) [reply en/docs/try-out/account-information-service-flow.md]: Fixed in 49adb6d8c05809680c301f250e674df1f369b3f0 & 2d265ec7996df3d487c08ddc78bbc2d6bac52487

File: en/docs/try-out/account-information-service-flow.md
Comment (isharailanga) [reply en/docs/try-out/account-information-service-flow.md]: Fixed in 49adb6d8c05809680c301f250e674df1f369b3f0

File: en/docs/try-out/account-information-service-flow.md
Comment (isharailanga) [reply en/docs/try-out/account-information-service-flow.md]: Fixed in 49adb6d8c05809680c301f250e674df1f369b3f0

File: en/docs/try-out/account-information-service-flow.md
Comment (isharailanga) [reply en/docs/try-out/account-information-service-flow.md]: Fixed in 49adb6d8c05809680c301f250e674df1f369b3f0

**Pull request merged**
